packages/investor/src/modules/copytrading-tables/components/open-trades-table.tsx

```diff
@@ -99,126 +38,40 @@ class OpenTradesTable extends React.PureComponent<
-                  </GVButton>
-                </Link>
-              </div>
-            </TableCell>
-            <TableCell className="managers-table__cell--username">
-              <ProfileAvatar
-                url={signalTrade.manager.avatar}
-                alt={signalTrade.manager.username}
-              />
-              <Link
-                to={{
-                  pathname: composeManagerDetailsUrl(signalTrade.manager.url),
-                  state: `/ ${title}`
-                }}
-              >
-                <GVButton variant="text" color="secondary">
-                  {signalTrade.manager.username}
-                </GVButton>
-              </Link>
-            </TableCell>
-            <TableCell>{moment(signalTrade.date).format()}</TableCell>
-            <TableCell>{signalTrade.symbol}</TableCell>
-            <TableCell>
-              <NumberFormat
-                value={formatValue(signalTrade.volume)}
-                displayType="text"
-                thousandSeparator=" "
-              />
-            </TableCell>
-            <TableCell>
-              <NumberFormat
-                value={formatValue(signalTrade.price)}
-                displayType="text"
-                thousandSeparator=" "
-              />
-            </TableCell>
-            <TableCell>
-              <Profitability
-                value={+formatPercent(signalTrade.profit)}
-                prefix={PROFITABILITY_PREFIX.SIGN}
-              >
-                <NumberFormat
-                  value={formatPercent(signalTrade.profit)}
-                  allowNegative={false}
-                  thousandSeparator=" "
-                  displayType="text"
-                  suffix=" %"
-                />
-              </Profitability>
-            </TableCell>
-            <TableCell>
-              <GVButton
-                className="gv-btn--no-padding close-btn"
-                color="secondary"
-                onClick={this.openConfirmPopup}
-              >
-                <CloseIcon />
-              </GVButton>
-              <ConfirmPopup
-                open={isConfirmPopupOpen}
-                onClose={this.closeConfirmPopup}
-                onApply={this.closeTrade(signalTrade.id, updateRow)}
-                header={t(
-                  "investor.copytrading-tables.close-trade-confirm.header"
-                )}
-                body={t(
-                  "investor.copytrading-tables.close-trade-confirm.body",
-                  {
-                    symbol: signalTrade.symbol,
-                    volume: formatValue(signalTrade.volume)
-                  }
-                )}
-                applyButtonText={t("buttons.confirm")}
-              />
-            </TableCell>
-          </TableRow>*/}
-          </>
+        renderBodyRow={(trade: OrderSignalModel, updateRow) => (
+          <TradeRow
+            key={trade.id}
+            trade={trade}
+            title={title}
+            //@ts-ignore
```

Comment:
You can use `UpdateRowFuncType` in your components for `update` props and set `row` as optional in `UpdateRowFuncType` (it really is such)